src/contraction/doc/contraction.rst

```diff
@@ -0,0 +1,167 @@
+..
+   ****************************************************************************
+    pgRouting Manual
+    Copyright(c) pgRouting Contributors
+
+    This documentation is licensed under a Creative Commons Attribution-Share
+    Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/
+   ****************************************************************************
+
+.. _contraction:
+
+Contraction
+===============================================================================
+
+The documentation provides explanation on graph contraction using small examples.
+
+level1 vertices have only one incoming edge.
+level2 vertices have one incoming edge and one outgoing edge.
+
+For simplicity all the edges in the examples have unit weight.
+
+.. _fig1:
+
+.. figure:: images/twoNodesOneEdge.png
+
+    *Graph 1: Directed, with two nodes and one edge*
+
+    G={V:{1,2},E:{(1,2)}}
```

Comment:
where:
 - V: is the set of vertices
 - E: is the set of edges
 - G is the graph